docs-sources/includes/_Actors.md

```diff
@@ -31,6 +73,9 @@ and session-information in memory).
 ```javascript
 exports.login = function (state, username, password, callback) {
   mage.auth.login(state, username, password, callback);
+
+  // if you are not use the auth module, auth can be done manually by calling
```

Comment:
It would be wiser to just tell people to use auth I think, or to actually provide a full-blown example on how to do this manually.